src/experiments.ts

```diff
@@ -134,6 +134,13 @@ export const ALL_EXPERIMENTS = experiments({
     default: true,
     public: false,
   },
+
+  compatiblemode: {
+    shortDescription: "Enable schema migrations in Compatible Mode",
+    fullDescription: "Enable schema migrations in Compatible Mode",
+    default: true,
```

Comment:
I believe `default: false` (or just remove this field entirely) since we're not launching yet.